### PR TITLE
Clarify claiming and update QR codes

### PR DESCRIPTION
**Pull request by johanstokking**
#### Summary

Clarify how to make a device claimable

Also updated the QR code instructions as we now support the final version of TR005

#### Screenshots

This one is a bit off, but that's because of the developer tools (I guess):

localhost_1313_devices_device-claiming_make-device-claimable_

#### Changes

- Update QR code instructions
- Clarify making a device claimable

#### Checklist

- [x] Scope: The referenced issue is addressed, there are no unrelated changes.
- [x] Run Locally: Verified that the docs build using `make server`, posted screenshots, verified external links. Test with `HUGO_PARAMS_SEARCH_ENABLED=true` if style changes will affect the search bar.
- [x] New Features Marked: Documentation for new features is marked using the `new-in-version` shortcode, according to the guidelines in [CONTRIBUTING](CONTRIBUTING.md).
- [x] Style Guidelines: Documentation obeys style guidelines in [CONTRIBUTING](CONTRIBUTING.md).
- [x] Commits: Commit messages follow guidelines in [CONTRIBUTING](CONTRIBUTING.md), there are no fixup commits left.

### CODE REVIEW

File: doc/content/devices/device-claiming/make-device-claimable.md [hunk 1]
```diff
@@ -5,49 +5,80 @@ weight: 1
...
-1. With root keys and claim authentication code
-- Follow the [Activating Devices on Cloud]({{< ref "/getting-started/cloud-hosted/tti-join-server/activate-devices-cloud-hosted" >}}) guide.
+In order to make a device claimable, the device needs to be added first in the application that has been authorized for claiming.
 
-2. With secure element claiming
-- Follow the [Claim ATECC608A Secure Elements]({{< ref "/devices/claim-atecc608a" >}}) and [Activating Devices on Cloud]({{< ref "/getting-started/cloud-hosted/tti-join-server/activate-devices-cloud-hosted" >}}) guides.
+If you already have devices in your application can be claimed, i.e. to transfer ownership, you can go ahead to the next section.
 
-## Generate QR code for claiming
+If you don't have a devices in the application yet, you can use the following ways to add a device:
```
Comment (nejraselimovic) [inline doc/content/devices/device-claiming/make-device-claimable.md:40]:
```suggestion
If you don't have a device in the application yet, you can add it by using one of the following ways:
```

File: doc/content/devices/device-claiming/make-device-claimable.md [hunk 2]
```diff
@@ -5,49 +5,80 @@ weight: 1
...
 
-After registering your devices, you need to generate QR codes for claiming.
+1. [Add a device]({{< relref "../adding-devices/" >}}) in {{% tts %}}
+2. [Register a device]({{< ref "/getting-started/cloud-hosted/tti-join-server/register-devices" >}}) on The Things Join Server (i.e. not in a Network Server or Application Server yet)
+3. [Import ATECC608A Secure Elements]({{< relref "../claim-atecc608a" >}}) on The Things Join Server. Be sure to check **Set claim authentication code**
```
Comment (nejraselimovic) [inline doc/content/devices/device-claiming/make-device-claimable.md:44]:
```suggestion
3. [Import ATECC608A Secure Elements]({{< relref "../claim-atecc608a" >}}) on The Things Join Server. Make sure to enable the **Set claim authentication code** option.
```

File: doc/content/devices/device-claiming/make-device-claimable.md [hunk 3]
```diff
@@ -5,49 +5,80 @@ weight: 1
...
 
-Learn how to [Generate QR Codes]({{< ref "/devices/generate-qr-code#generate-qr-code-for-claiming" >}})
+## Claiming Settings
 
-## Authorize Claiming
-  
-Replace `<app-id>` with the **Application ID** of the application that you created in prerequisites, and run the following command in the CLI:
+When your device is added in {{% tts %}}, you can go ahead and configure claiming settings. This is comprised of a claim authentication code and a validity window. The claim authentication code is a secret value. The validity window is an optional start and end date that the claim authentication code can be used.
```
Comment (nejraselimovic) [inline doc/content/devices/device-claiming/make-device-claimable.md:48]:
```suggestion
When your device is added in {{% tts %}}, you can go ahead and configure claiming settings. This is comprised of a claim authentication code and a validity window. The claim authentication code is a secret value. The validity window is an optional start and end date on which the claim authentication code can be used.
```

File: doc/content/devices/device-claiming/make-device-claimable.md [hunk 4]
```diff
@@ -5,49 +5,80 @@ weight: 1
...
 
+{{< tabs/container "Console" "CLI" >}}
+
+{{< tabs/tab "Console" >}}
+
+Go to your application in {{% tts %}} Console. Go to **End devices** in the left menu and find the device you want to configure for claiming.
+
+In the **Claiming** tab, you find the claiming settings:
```
Comment (nejraselimovic) [inline doc/content/devices/device-claiming/make-device-claimable.md:56]:
```suggestion
Find the claiming settings in the **Claiming** tab:
```

File: doc/content/devices/device-claiming/make-device-claimable.md [hunk 5]
```diff
@@ -5,49 +5,80 @@ weight: 1
...
+
+{{< figure src="../claiming-settings.png" alt="Claiming settings" >}}
+
+You can edit the claim authentication code and validity window here. To save, click **Save changes**.
```
Comment (nejraselimovic) [inline doc/content/devices/device-claiming/make-device-claimable.md:60]:
```suggestion
To save the claiming settings you edited, click **Save changes**.
```

File: doc/content/devices/device-claiming/make-device-claimable.md [hunk 6]
```diff
@@ -5,49 +5,80 @@ weight: 1
...
+
+To disallow claiming, click **Delete claim authentication code**.
+
+{{< /tabs/tab >}}
+
+{{< tabs/tab "CLI" >}}
+
+To configure the claiming using the command-line interface:
```
Comment (nejraselimovic) [inline doc/content/devices/device-claiming/make-device-claimable.md:68]:
```suggestion
To configure claiming using the CLI, use the following command:
```

File: doc/content/devices/generate-qr-code/_index.md [hunk 7]
```diff
@@ -44,30 +44,32 @@ The formats show the fields of the end device that are used in the QR code.
...
+{{< figure src="qr-identification.png" alt="Device QR Code for Identification" >}}
+
+</details>
+
 This saves the QR code to the current directory with the device ID as file name, in PNG format with a default size of 300 pixels. Use `--folder` and `--size` to change the save location and image size.
 
 ## Generate QR Code for Claiming
 
-{{< figure src="qr-claiming.png" alt="Device QR Code for Claiming" class="plain float" >}}
+Device claiming is a mechanism to transfer devices securely from one application to another. For example, from a device maker to a device owner, or transferring ownership to new device owner. [Learn how to make a device claimable]({{< relref "../device-claiming/make-device-claimable" >}}).
```
Comment (nejraselimovic) [inline doc/content/devices/generate-qr-code/_index.md:63]:
```suggestion
Device claiming is a mechanism to transfer devices securely from one application to another. For example, from a device maker to a device owner, or transferring ownership to new device owner. Learn how to [make a device claimable]({{< relref "../device-claiming/make-device-claimable" >}}).
```

File: doc/content/devices/generate-qr-code/_index.md [hunk 8]
```diff
@@ -44,30 +44,32 @@ The formats show the fields of the end device that are used in the QR code.
...
 
-Device claiming is a mechanism to transfer devices securely from one application to another. For example, from a device maker to a device owner, or transferring ownership to new device owner.
-
-When you create a device, you can generate a claim authentication code by specifying the `--with-claim-authentication-code` flag. You can also set a claim authentication code via CLI:
+When a device is claimable (it contains a claim authentication code), you can use the same command to generate a QR code:
```
Comment (nejraselimovic) [inline doc/content/devices/generate-qr-code/_index.md:65]:
```suggestion
When a device is claimable (it contains a claim authentication code), you can use the same command as above to generate a QR code:
```

**Review by nejraselimovic — APPROVED** [review]

**Comment by johanstokking** [comment]
@nejraselimovic oops, `gh` CLI was showing an empty body of the approved review. Will address your reviews and push to `master`.

**Comment by johanstokking** [comment]
Done, see c6981870

**Pull request merged**